.github/workflows/release.yml

```diff
@@ -1,23 +1,30 @@
-name: rc release
+name: release
 
 on:
+  schedule:
+    - cron: "0 8 * * *"
   workflow_dispatch:
     inputs:
-      force:
-        description: 'force release even if no update(set empty if not)'
-        required: false
-        default: '--force-publish'
-      stable:
-        description: 'promote to stable version(true or false)'
-        required: false
-        default: 'false'
+      version:
+        description: 'which version to bump(major, minor, patch, premajor, preminor, prepatch, prerelease)'
```

Comment:
Remove premajor, preminor, prepatch from the description to avoid confusion? Seems they are not used in defined conditions.

Reply:
preminor & prepatch will be used when bump version in dev & hotfix branch.